packages/odyssey-react/src/components/Box/_box-root.scss

```diff
@@ -34,103 +34,7 @@
   }
   /* stylelint-enable selector-max-universal */
 
-  // Huge selector list layout reset
   /* stylelint-disable selector-max-type */
-  div,
-  span,
-  applet,
-  object,
-  iframe,
-  h1,
-  h2,
-  h3,
-  h4,
-  h5,
-  h6,
-  p,
-  blockquote,
-  pre,
-  a,
-  abbr,
-  acronym,
-  address,
-  big,
-  cite,
-  code,
-  del,
-  dfn,
-  em,
-  img,
-  ins,
-  kbd,
-  q,
-  s,
-  samp,
-  small,
-  strike,
-  strong,
-  sub,
-  sup,
-  tt,
-  var,
-  b,
-  u,
-  i,
-  center,
-  dl,
-  dt,
-  dd,
-  ol,
-  ul,
-  li,
-  fieldset,
-  form,
-  label,
-  legend,
-  table,
-  caption,
-  tbody,
-  tfoot,
-  thead,
-  tr,
-  th,
-  td,
-  article,
-  aside,
-  canvas,
-  details,
-  embed,
-  figure,
-  figcaption,
-  footer,
-  header,
-  hgroup,
-  menu,
-  nav,
-  output,
-  ruby,
-  section,
-  summary,
-  time,
-  mark,
-  audio,
-  video {
-    margin-block: 0;
-    margin-inline: 0;
-    padding-block: 0;
-    padding-inline: 0;
-    border: 0;
-    font: inherit;
-    font-size: 100%;
-    vertical-align: baseline;
-  }
-
-  button,
-  input {
-    font: inherit;
-    font-size: 100%;
-  }
-
   // HTML5 display-role reset for older browsers
```

Comment:
which older browsers? I think we should avoid styling raw HTML tag children within Box in this way as much as possible.

Reply:
IE11 - noted elsewhere, these might be safe to drop if we're done with that level of support.

Reply:
If we keep these, we'll want to ensure they apply to `.root` and not just children of `.root`.

Reply:
Removing should be ok `main` is the only element not supported and we don't style it
https://caniuse.com/html5semantic